proto/iotics/api/input.proto

```diff
@@ -72,6 +79,84 @@ message DeleteInputResponse {
 }
 // ---------------------------------------
 
+// CreateInputRequest describes a created input.
+message CreateInputRequest {
+  // CreateInputRequest payload.
+  message Payload {
+    // Input identifier string representation (simple string) (unique within the scope of a twin identifier's input set)
+    string id = 1;
+  }
+
+  // Arguments describes the mandatory arguments to identify the twin owning this input
+  message Arguments {
+    // Identifier of the twin owning this input
+    TwinID twinId = 1;
+  }
+
+  // CreateInputResponse headers
+  Headers headers = 1;
+  // CreateInputRequest arguments
+  Arguments args = 2;
+  // CreateInputResponse payload
+  Payload payload = 3;
+}
+// ---------------------------------------
+
+// CreateInputResponse describes a created input.
+message CreateInputResponse {
+  // CreateInputResponse payload.
+  message Payload {
+    // Created input
+    InputID inputId = 1;
+  }
+
+  // CreateInputResponse headers
+  Headers headers = 1;
+  // CreateInputResponse payload
+  Payload payload = 2;
+}
+// ---------------------------------------
+
+// UpdateInputRequest is used to update an input.
+message UpdateInputRequest {
+  // UdpateInputRequest payload used to update one or more field
+  message Payload {
+    // StoreLast dictates if we should store the last received message for this input
```

Comment:
store-last only applies to Feeds so this can be taken out.

The idea of store last for feeds is: Allow to receive the last-shared sample for slowly-updating Feeds. In the future the plan is to fold this into a proper persistence feature.
For inputs, because we can receive samples from anyone - keeping just the last one doesn't seem to be that useful. (But again maybe with a persistence feature in place it would be better and could store N.)

Reply:
Ah okay, it did seem like a field that might not make sense, thanks for the clarity